packages/cli/src/commands/doctor/healthchecks/python.ts

```diff
@@ -0,0 +1,55 @@
+import versionRanges from '../versionRanges';
+import {doesSoftwareNeedToBeFixed} from '../checkInstallation';
+import {logManualInstallation} from './common';
+import {HealthCheckInterface} from '../types';
+
+import download from '../../../tools/downloader';
+import {updateEnvironment} from '../../../tools/variables';
+import {join} from 'path';
+import {Ora} from 'ora';
+import {executeCommand} from '../../../tools/executeWinCommand';
+
+export default {
+  label: 'Python',
+  getDiagnostics: async ({Languages}) => ({
+    needsToBeFixed: doesSoftwareNeedToBeFixed({
+      version:
+        (Languages && Languages.Python && Languages.Python.version) || '',
```

Comment:
I need to recheck that `Languages && Languages.Python` is still needed now that envinfo always shows the missing ones.

Reply:
I've removed those checks as they aren't needed anymore 🎉